languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -178327,5 +178327,17 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="de flagrant|d'indéniable|d'incontestable|de sûr|de certain">Il n'y a rien <marker>d'évident</marker>.</example>
             </rule>
         </rulegroup>
+            <rule id="REP_CERTES" name="certes (style)" min_prev_matches="1" tags="picky">
```

Comment:
So I checked, the category is 
`category id="REPETITIONS_STYLE" name="repetitions style" type="style">` so it's indeed a style rule :p